settings.gradle

```diff
@@ -9,6 +9,7 @@ dependencyResolutionManagement {
     repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
     repositories {
         google()
+        jcenter()
```

Comment:
It seems like `jcenter` is deprecated. Just wondering, what did you need this for?

Reply:
I believe I added this when following a guide to set up the map functionality, but I am unsure what it is used for. The guide may have been outdated